package.json

```diff
@@ -18,7 +18,9 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "prettier ./src/**/*.js --check",
+    "format": "prettier ./src/**/*.js --write "
```

Comment:
Olga has this included in the scripts on the BE which might be helpful to add here?
"prettier-watch": "onchange \"**/*\" -- prettier --write --ignore-unknown {{changed}}"

it will make formatting happen automatically whenever changes are made to any file. We can ask Dylan if there is a reason not to use it though - I'm not sure what best practice is here.